std/format.d

```diff
@@ -3040,19 +3040,46 @@ if (isInputRange!T)
             static if (is(StringTypeOf!T))
             {
                 auto s = val[0 .. f.precision < $ ? f.precision : $];
+
+                size_t width;
+                if (f.width > 0)
+                {
+                    // ASCII dstring & string can be aligned w/o graphemeStride
+                    bool onlyAscii;
+                    static if (!is(Unqual!(ElementEncodingType!T) == wchar))
+                    {
+                        onlyAscii = true;
+                        for (size_t i; i < s.length; i++)
+                            if (s[i] > 0x7F)
+                        {
```

Comment:
Style nitpick: please add braces around the loop body and an extra indent for the if-statement body. I know it's annoying to have another level of nesting for just a single statement, but I'd rather we stick to the Phobos style guidelines for consistency's sake.

Reply:
Okay done, but i'm sure i've already been merged in Phobos even if the code included an usage of this particular brace style.